lms/static/sass/course/courseware/_sidebar.scss

```diff
@@ -48,7 +48,7 @@
           border-radius: 0;
           box-shadow: none;
           @include padding-left(19px);
-          color: $link-color;
+          color: $blue-on-gray;
```

Comment:
this breaks theming because link color is meant to be general - once you change the blue above, you can revert this to $link-color